templates/long.yaml

```diff
@@ -32,3 +32,11 @@
 
 - name: Test Project - Empty
   type: Project
+
+- name: Test Team
+  type: Team
+  can_public_join: false
+  description: Test team description
+  members:
+  - principal_id: 3426116
```

Comment:
I originally looked at the Synapse REST [docs](https://rest-docs.synapse.org/rest/POST/membershipInvitation.html) for the team invitation API, which only offers `inviteeId` (_i.e._ principal ID) and `inviteeEmail` as parameters. 

I wondered if the `Synapse.invite_to_team()` [method](https://python-docs.synapse.org/build/html/index.html#synapseclient.Synapse.invite_to_team) in the Python client supported Synapse user names, which it does. 

I think we should eventually support user names (and team names in ACLs), but this ultimately feels like a task for the short template, whereby the expansion code would query Synapse for the principal IDs and drop them in the template. Do others agree?

Reply:
I agree that we should eventually support usernames, but we don't have to currently.

Reply:
Cool, we're on the same page.